sql/hive-thriftserver/src/main/java/org/apache/hive/service/auth/HttpAuthUtils.java

```diff
@@ -89,7 +89,7 @@ public static String getKerberosServiceTicket(String principal, String host,
    * @param clientUserName Client User name.
    * @return An unsigned cookie token generated from input parameters.
    * The final cookie generated is of the following format :
-   * cu=<username>&rn=<randomNumber>&s=<cookieSignature>
+   * {@code cu=<username>&rn=<randomNumber>&s=<cookieSignature>}
```

Comment:
This is java code. So, `@code` should be fine. This also seems not exposed to the documentation anyway.